Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/priority_clock.cpp

```diff
@@ -0,0 +1,105 @@
+#include "priority_clock.hpp"
+
+
+std::string priority_clock::get_unique_id(ClassificationItem& item)
+{
+	std::string id = item.name;
+	
+	id.append(std::to_string(item.id));
+	return id;
+}
+
+bool priority_clock::check_cooldown(std::string id, int time_dif)
+{
+	auto F = cooldowns.find(id);
+	auto time = clock();
+
+	SPDLOG_INFO("Item Finding {}, looking for time {}", id, time_dif);
```

Comment:
Wouldn't 'Finding Item {}' be more descriptive? 😄

Reply:
And in the next SPDLOG statement 😀

Reply:
Fixed.